README.md

```diff
@@ -13,17 +13,7 @@ service.
 
 ## Getting Started
 
-* Try the [sample app](Sample).
-  * Using [CocoaPods](https://cocoapods.org):
-    ```
-    pod try GoogleSignIn
-    ```
-  * Using [Swift Package Manager](https://swift.org/package-manager/):
-    ```
-    git clone https://github.com/google/GoogleSignIn-iOS
-    ```
-    ```
-    open GoogleSignIn-iOS/Sample/SignInSample.xcodeproj
```

Comment:
How about providing an 'open' command for both projects for quick copying?

Reply:
The READMEs for each sample project, linked in the bullet above, have an `open` command.